_data/resources.yml

```diff
@@ -331,3 +331,6 @@
     - url: http://tenon.io/
       title: "Accessibility Testing API"
       description: "By Tenon.io"
+    - url: https://packagecontrol.io/packages/WAI-ARIA
```

Comment:
@indrekpaas and @nschonni Which link should be used here? The one in this PR or the one mentioned in #460 (https://github.com/indrekpaas/sublime-wai-aria)? I understand that the page on Package Control refers and links to this GitHub page, but which one is better (and why)?